core/arch/arm/arm.mk

```diff
@@ -18,3 +18,10 @@ core-platform-cppflags += $(arm32-platform-cppflags)
 core-platform-cflags += $(arm32-platform-cflags)
 core-platform-aflags += $(arm32-platform-aflags)
 endif
+
+ifeq ($(CFG_WITH_PAGER),y)
+ifneq ($(CFG_DISABLE_CONCURRENT_EXEC),y)
```

Comment:
> In case 3), I would raise an error anyway, as it is now. Nobody looks at warnings.

Yep, it was my initial reasoning too. Let's keep it then.

> We may change the name from force to check_or_set (well, I'm not very good at naming).

I have no better suggestion than `force`, which is not so bad actually because *if* the build proceeds, then you are certain that the value is the one supplied.

> Can the reason be optional?

Sure, no problem. All make macros can take a variable number of arguments.